config/controller.yaml

```diff
@@ -91,6 +91,8 @@ spec:
         # If you are changing these names, you will also need to update
         # the controller's Role in 200-role.yaml to include the new
         # values in the "configmaps" "get" rule.
+        - name: CONFIG_DEFAULTS_NAME
+          value: config-defaults
```

Comment:
Interesting, why wasn't this here before ? (or why is it needed now ?)
/cc @afrittoli

Reply:
We have a fallback of "config-defaults" which is returned [here](https://github.com/tektoncd/pipeline/blob/0b345b56bda47c4542519102527889f7d0777a33/pkg/apis/config/default.go#L54-L59), which is likely why it's never caused problems not being here before.

I think I added this while Jerop and I were pairing on these changes. This is just a drive-by fix and not a strict requirement of this PR (I don't think - @jerop I might be misremembering?)

Reply:
yes, that's right, we thought to fix it but not needed in this PR

why it's needed in general is that users may need to give the config maps custom names to avoid collisions with other services, as described in issue https://github.com/tektoncd/pipeline/issues/1805

Reply:
The name of config maps used in the config store is configurable via environment variable today: https://github.com/tektoncd/pipeline/blob/93a4e5fa8d4ec8f7d099a5e9ad80bf79405c1507/pkg/apis/config/default.go#L54-L59 . Having the environment variable in the controller.yaml makes it easier for users who may want to change the default name. It's nice to have, but not related to this PR.
I don't mind having it in this PR though.